build.sh

```diff
@@ -29,9 +30,9 @@ dist() {
    -e "s|@RPM_RELEASE@|$RPM_RELEASE|g" \
    -e "s|@PACKAGE_NAME@|$PACKAGE_NAME|g" \
    -e "s|@PACKAGE_VERSION@|$PACKAGE_VERSION|g" \
-   < ovirt-ansible-hosted-engine-setup.spec.in > ovirt-ansible-hosted-engine-setup.spec
+   < ovirt_ansible_hosted_engine_setup.spec.in > ovirt_ansible_hosted_engine_setup.spec
```

Comment:
Still not 100% sure about the rpm name change. Everything else is already ovirt-